planning/project-plan.md

```
@@ -0,0 +1,29 @@
+
+- Project explanation
+- Contribution roadmap
+- Beginner-friendly tasks
+- First PR checklist
+
+## Future Features
+
+- Good First Issue Finder
+- Tech Stack Detection
+- Difficulty Estimation
+- Setup Guide
```

markdownlint-cli2 (0.23.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)